CMakeLists.txt

```diff
@@ -96,6 +96,9 @@ install(
 
 if(${Python_INTERFACE})
   message("-- Copying python examples")
+  configure_file(${PROJECT_SOURCE_DIR}/src/gm2_python_interface.py.in 
+                 ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/gm2_python_interface.py 
```

Comment:
The idea is great, but I think this file is missing.
Could this also be used in the tests?